content/en/docs/concepts/storage/volumes.md

```diff
@@ -1007,300 +725,47 @@ backed by tmpfs (a RAM-backed filesystem) so they are never written to
 non-volatile storage.
 
 {{< note >}}
-You must create a Secret in the Kubernetes API before you can use it.
-{{< /note >}}
-
-{{< note >}}
-A container using a Secret as a [`subPath`](#using-subpath) volume mount will not
-receive Secret updates.
+A container using a Secret as a [`subPath`](#using-subpath) volume mount doesn't receive Secret updates.
 {{< /note >}}
 
 For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
 
-### storageOS {#storageos}
 
-A `storageos` volume allows an existing [StorageOS](https://www.storageos.com)
-volume to mount into your Pod.
+## Storage plugin interfaces
```

Comment:
The "storage plugin" section earlier in this doc seems to duplicate some of this section?

Reply:
I intended that earlier section to be a quick overview of the options, and this section to be a bit more detailed.

Reply:
If this section is only for out of tree plugin extensions should we call this something like "External plugin extensions"

Reply:
In terms of ordering, should we put this before or after listing all the in-tree types? fit seems a little strange that we're talking about external plugins in between 2 in-tree plugin sections